server/src/routes/assistantRoutes.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const {
+    sendMessageHandler
+} = require('../controllers/assistantController');
+
+const router = express.Router();
+
+router.post('/sendMessage', sendMessageHandler);
```

Comment:
So the call will be 'api/openai/sendMessage' under the group of "GPT Assistance"

Can you make it so that it is 'api/openai/assistant/sendMessage' and the group is called "GPT Assistants"